bigtable/client/client_options.h

```diff
@@ -81,15 +82,56 @@ class ClientOptions {
   }
 
   /**
-   * Set the grpclb fallback timeout (in ms) for the channel.
+   * Set the grpclb fallback timeout with the timestamp [@p fallback_timeout)
+   * for the channel.
+   *
+   * This function accepts any instantiation of 'std::chrono::duration<>' for
+   * @p duration parameter convert it into milliseconds and pass it to
+   * channel_arguments. For example:
+   *
+   * @code
+   * using namespace std::chrono_literals; // C++14
+   * bigtable::ClientOptions::SetGrpclbFallbackTimeout(std::chrono::milliseconds(5000))
+   * bigtable::ClientOptions::SetGrpclbFallbackTimeout(std::chrono::seconds(5))
+   * @endcode
+   *
+   * The fallback_timeout must not be empty and it should be within the range
+   * of int32. The code will throw exception std::out_of_range if range goes
+   * outside of int32.
+   *
+   * @tparam Rep a placeholder to match the Rep tparam for @p fallback_timeout,
+   *     the semantics of this template parameter are documented in
+   *     `std::chrono::duration<>` (in brief, the underlying arithmetic type
+   *     used to store the number of ticks), for our purposes it is simply a
+   *     formal parameter.
+   * @tparam Period a placeholder to match the Period tparam for @p
+   * fallback_timeout
```

Comment:
Please indent the paragraph correctly.  If `fallback_timeout` does not fit in the previous line it should be indented by 4 spaces and the rest of the paragraph needs to be reflowed.

Reply:
This is still pending, the rest looks good, please fix so we can merge your changes.